readme.md

```diff
@@ -90,6 +90,17 @@ $({stdio: 'inherit'}).sync`echo rainbows`;
 //=> 'rainbows'
 ```
 
+#### With results from `$` or `$.sync`
```

Comment:
Open to suggestions here:
- `` Interpolating results from `$` or `$.sync` ``
- `` Interpolating results from previous `$` or `$.sync` ``
- etc.

Reply:
This looks good to me :+1: